.circleci/config.yml

```diff
@@ -51,7 +46,22 @@ jobs:
     environment:
       GO111MODULE: "on"
 
-# packet
+  # code analysis
+  static-code-analysis:
```

Comment:
@nickolaev I've added job with `static-code-analysis` and got strange result: https://circleci.com/gh/networkservicemesh/networkservicemesh/46875?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link
```
Received 'killed' signal
```
Are you know why this happens?